apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/verify_first_vote.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "base"
+
+module ActionTracking
+  module Strategies
+    class VerifyFirstVote < Base
+      def on_start_execution
+        {
+          proposalsVotedOn: agora_delegate["proposalsVotedOn"].to_i
```

Comment:
any reason to use camelCase here? 🤔 I think we have as a standard using snake_case in these hashes' keys

Reply:
No reason at all, I will change!!